projects/envoy/build.sh

```diff
@@ -119,6 +119,11 @@ do
   mkdir -p "${CORPUS_UNTAR_PATH}"
   tar -C "${CORPUS_UNTAR_PATH}" -xvf bazel-bin/"${t}"_corpus_tar.tar
   TARGET_BASE="$(expr "$t" : '.*/\(.*\)_fuzz_test')"
+  # There may be *.dict files in this folder that need to be moved into the OUT dir.
+  for dictionary in "${CORPUS_UNTAR_PATH}"/*.dict; do
+    [ -f "$dictionary" ] || continue
```

Comment:
Why is this needed? When would we hit something matching `*.dict` that isn't a regular file?

Reply:
Hm, just to make sure there are dictionaries. if there's not `*.dict` then mv will log a cannot stat. I think I'll just make it a one-liner find with -exec mv.